static/js/components/email/EmailCompositionDialog.js

```diff
@@ -78,6 +79,9 @@ export default class EmailCompositionDialog extends React.Component {
   constructor (props: EmailDialogProps) {
     super(props);
     this.state = editorStateFromProps(props);
+    if (R.isNil(this.state.editorState)) {
```

Comment:
@aliceriot Do you have a suggestion how to update `editorStateFromProps` to set this? I see the `S.map` but I'm not sure how you convert a nothing into a different value